components/zcash_address/src/kind/unified.rs

```diff
@@ -538,12 +580,13 @@ pub(crate) mod private {
                 Err(ParseError::OnlyTransparent)
             } else {
                 // All checks pass!
-                Ok(Self::from_inner(items))
+                Ok(Self::from_inner(revision, items))
             }
         }
 
         fn parse_internal<T: Into<Vec<u8>>>(hrp: &str, buf: T) -> Result<Self, ParseError> {
-            Self::parse_items(hrp, buf).and_then(Self::try_from_items_internal)
+            Self::parse_items(hrp, buf)
+                .and_then(|(revision, items)| Self::try_from_items_internal(revision, items))
```

Comment:
I _think_ this line should work fine as it was before?

Reply:
A tuple can't be converted to an argument pair.